javascript-typing-tutor/main.js

```diff
@@ -0,0 +1,41 @@
+var $char = document.querySelectorAll('span');
+var $modal = document.querySelector('.modal');
+var $stats = document.querySelector('.stats');
+var $buttonYes = document.querySelector('.button-yes');
+var $buttonNo = document.querySelector('.button-no');
+var i = 0;
+var correct = 0;
+var incorrect = 0;
+var totalType = 0;
+var accuracy = 0;
```

Comment:
Instead of declaring these here, just declare them right where you need them (`totalType`, `accuracy`).

Reply:
Okay :)